backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandra.java

```diff
@@ -118,12 +120,12 @@ private void grantPermissionToTestingUser(Cluster privilegedCluster, String keys
 
     @SuppressWarnings("resource")
     public DockerCassandra() {
-        this("cassandra_3_11_3", AdditionalDockerFileStep.IDENTITY);
+        this("cassandra_3_11_3-"+ UUID.randomUUID().toString(), AdditionalDockerFileStep.IDENTITY);
     }
 
     public DockerCassandra(String imageName, AdditionalDockerFileStep additionalSteps) {
         client = DockerClientFactory.instance().client();
-        boolean doNotDeleteImageAfterUsage = false;
+        boolean doNotDeleteImageAfterUsage = true;
```

Comment:
we should rename the variable to `deleteImageAfterUsage`

Reply:
this chance was reverted afterwards in a vaguely related commit. I have rewritten the history to clean up my mess. 
Should I still rename the variable?

Reply:
If you revert it, no

Reply:
changes to this variable were reverted, I'll resove this :)